packages/clerk-js/src/ui/new/index.tsx

```diff
@@ -37,13 +37,15 @@ export class UI {
     this.register('SignIn', {
       type: 'component',
       load: () =>
+        // eslint-disable-next-line import/no-unresolved
         import(/* webpackChunkName: "rebuild--sign-in" */ '@clerk/ui/sign-in').then(({ SignIn }) => ({
           default: SignIn,
         })),
     });
     this.register('SignUp', {
       type: 'component',
       load: () =>
+        // eslint-disable-next-line import/no-unresolved
```

Comment:
❓ Why we are disabling the eslint rule here, seems that it's not related to the PR